scripts/cargo/bench/list-crates-with-bench.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+set -eu
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+WS_CRATES=$($SCRIPT_DIR/get-workspace-crates.sh)
+echo "$WS_CRATES" | jq -r -c '[.[] | select(.targets[] | .kind | contains(["bench"])) | .name] | sort | unique' | jq -r -c '[.[] | select(. != "napi" and . != "wasm")]'
```

Comment:
I don't know enough to say there's a problem here, but it seems suspicious that you're piping `jq` to `jq`.

Reply:
🤣 It was copied from https://github.com/swc-project/swc/blob/15ea059712f6726d52b6304870f337a93272bb53/scripts/cargo/get-workspace-crates-json.sh (very old script) and https://github.com/swc-project/swc/blob/15ea059712f6726d52b6304870f337a93272bb53/scripts/bench/list-crates-with-bench.sh (relatively young script)

I decided to simply call the old script while writing `list-crates-with-bench.sh `

Reply:
Ah, if you mean `jq | jq`, it's because while triaging it worked :trollface: